src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -30,9 +51,20 @@ private void initUI() {
         pack();
     }
 
+    private class DefaultPanel extends JPanel {
```

Comment:
Is defining a new class necessary for now? What common features between our panels you are trying to capture?